src/Symfony/Component/Messenger/ConfiguredMessage.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger;
+
+/**
+ * Wrapped-message holding configuration.
+ *
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ *
+ * @experimental in 4.1
+ */
+final class ConfiguredMessage
```

Comment:
Could you add some basic tests for this class? Especially for the `getMessageFor` and `fromMessage` ones?

Reply:
Done